src/Config.ts

```diff
@@ -18,23 +18,31 @@
 import { Node, Link, NodeAttrs, LinkAttrs } from './Topology'
 import Tools from './Tools'
 
-const WEIGHT_NONE = 0
-const WEIGHT_FABRIC = 10
-const WEIGHT_PHYSICAL = 13
-const WEIGHT_BRIDGES = 14
-const WEIGHT_PORTS = 15
-const WEIGHT_VIRTUAL = 17
-const WEIGHT_NAMESPACE = 18
-const WEIGHT_VMS = 19
-const WEIGHT_CONTAINERS = 20
-const WEIGHT_K8S_FEDERATION = 100
-const WEIGHT_K8S_CLUSTER = 101
-const WEIGHT_K8S_NODE = 102
-const WEIGHT_K8S_NAMESPACE = 103
-const WEIGHT_K8S_POD = 104
-const WEIGHT_K8S_CONTAINER = 105
-const WEIGHT_K8S_SERVICE = 106
-const WEIGHT_K8S_OTHER = 120
+const SHOW_DEBUG = "True"
```

Comment:
nit: just SHOW_DEBUG = true

And I would prefer to have it set to false as default

Reply:
done